CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
```

Comment:
Hmm, `Adds` sounds a little odd. 

Maybe: `Adds ability to run shard rebalancer in the background` or such?